apollo-ios-pagination/Sources/ApolloPagination/GraphQLQueryPager.swift

```diff
@@ -155,7 +155,7 @@ public class GraphQLQueryPager<InitialQuery: GraphQLQuery, PaginatedQuery: Graph
           } else {
             shouldUpdate = true
           }
-          let variables = initialQuery.__variables?.values.compactMap { $0._jsonEncodableValue?._jsonValue } ?? []
+          let variables = nextPageQuery.__variables?.values.compactMap { $0._jsonEncodableValue?._jsonValue } ?? []
```

Comment:
@Iron-Ham - this seems like a reasonable and correct change but do you have an opinion on it? _I can't seem to request you as a reviewer and I'm not familiar enough with this code yet._

Reply:
The fix as written is correct. It is also addressed as part of: https://github.com/apollographql/apollo-ios-dev/pull/80

Reply:
Got it. Are you opposed to merging this so the author can continue targeting the `main` branch while #80 is still in development?

Reply:
I don't mind rebasing my branch at all; #80 is ready for review, but not as easily merged as this. 😜